Source/WebKit/UIProcess/WebAuthentication/fido/U2fAuthenticator.cpp

```diff
@@ -109,8 +109,11 @@ void U2fAuthenticator::issueSignCommand(size_t index)
         return;
     }
     auto u2fCmd = convertToU2fSignCommand(requestData().hash, requestOptions, requestOptions.allowCredentials[index].id, m_isAppId);
+    if (UNLIKELY(!u2fCmd)) {
+        ASSERT_NOT_REACHED();
```

Comment:
This seems like a somewhat valid state based on `convertToU2fSignCommand`. If a request requires UV and you stick a U2F key in, you'll hit this. 

Let's not ASSERT_NOT_REACHED here and instead log something like
```
U2F_RELEASE_LOG("issueSignCommand: request not convertible to U2F.");
observer()->authenticatorStatusUpdated(WebAuthenticationStatus::KeyStoreFull);
```

and return.

Reply:
I'll filed https://bugs.webkit.org/show_bug.cgi?id=270877 as an enhancement to show a message here as well, but that requires changes to other projects as well. For this PR, I think just the logging is great.

Reply:
also let's do something similar for convertToU2fRegisterCommand (same problem)